sites/friday/src/get-started/nextjs/add-database.md

```diff
@@ -0,0 +1,228 @@
+      mariadb: "mariadb:mysql"
+```
+
+Add into your ``.environment`` file the following lines:
+```shell {location=".environment"}
+export RELATIONSHIPS_JSON="$(echo $PLATFORM_RELATIONSHIPS | base64 --decode)"
+
+# Set database environment variables
+export DB_HOST="$(echo $RELATIONSHIPS_JSON | jq -r '.mariadb[0].host')"
+export DB_PORT="$(echo $RELATIONSHIPS_JSON | jq -r '.mariadb[0].port')"
+export DB_DATABASE="$(echo $RELATIONSHIPS_JSON | jq -r '.mariadb[0].path')"
+export DB_USERNAME="$(echo $RELATIONSHIPS_JSON | jq -r '.mariadb[0].username')"
+export DB_PASSWORD="$(echo $RELATIONSHIPS_JSON | jq -r '.mariadb[0].password')"
+```
+
+{{< note >}}
+This additional `.environment` file is located at the root of your source code. During deployment of your {{% vendor/name %}} project, this file will override existing `.env` environment variables with {{% vendor/name %}} specific ones.
+{{< /note >}}
+
+Commit your change:
+```bash {location="Terminal"}
+git add  .environment .{{% vendor/cli %}}/config.yaml && git commit -m "adding MariaDB database service"
+{{% vendor/cli %}} push
+```
+
+{{< note >}}
+{{% vendor/name %}} will now read your configuration files, and begin building your application image. **Your push
+will fail**; don't worry, this is expected. At this point {{% vendor/cli %}} is not aware of the resources
+our new service needs. We need to tell it what kind of CPU, Memory, and disk to assign to the service container.
+
+Please complete [Configure resources for your service](#configure-resources-for-your-service) step
+{{< /note >}}
+
+## 3. Configure resources for your service
+When you push a new service to your project for the first time, the deployment will fail, and you will need to allocate resources to the service container.
+
+Back in your terminal, run:
+
+```shell
+{{% vendor/cli %}} resource:set --size mariadb:0.5 --disk mariadb:512
+  Resource configuration for the project My Express App 0926 (12345azerty), environment feat-database (type: development):
+  +------------------+------+-----+-------------+-----------+-----------+
+  | App or service   | Size | CPU | Memory (MB) | Disk (MB) | Instances |
+  +------------------+------+-----+-------------+-----------+-----------+
+  | app              | 0.1  | 0.1 | 384         | N/A       | 1         |
+  | mariadb          | 0.5  | 0.5 | 1408        | 512       | 1         |
+  +------------------+------+-----+-------------+-----------+-----------+
+
+Summary of changes:
+  Service: mariadb
+    CPU: 0.5
+    Memory: 1408 MB
+    Disk: 512 MB
+
+Are you sure you want to continue? [Y/n] y
+```
+
+## 4. Configure your Express application to use this new database
```

Comment:
```suggestion
## 4. Configure your NextJS application to use this new database
```

Reply:
I change this file extension from `.md` to `.todo` to not display it but keep the file... maybe i can remove it from the source code if you prefer?
And yes, it was a copy/paste from Express and i didn't review it.